contact.html

```diff
@@ -12,14 +12,18 @@
     <div class="main-container">
         <header class="header">
             <div class="logo">
-                <div class="name">EDAH ŠAHINOVIĆ</div>
+                <div class="name">
+                    <a href="/">EDAH ŠAHINOVIĆ</a>
+                </div>
             </div>
             <div class="navigation">
                 <nav>
                     <ul>
-                        <li><a href="/">ABOUT ME</a></li>
                         <li><a href="cv.html">CV</a></li>
                         <li><a href="projects.html">PROJECTS</a></li>
+                        <li><a href="certificates.html" class="certificates">
+                            CERT<span class="cert-full">IFICATES</span>
```

Comment:
Cheap way to get more space. Use "CERT" on smaller devices, while also using the "IFICATES" on normal devices.